loops.js

```diff
@@ -33,6 +42,13 @@
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over. Google daarom eens naar 'mdn webdocs remainder operator'.
 // ==========================================
 
+for(let i = 0; i<15; i++) {
+
+if(i%2)
```

Comment:
Met een if-statement check je of de conditie waar of niet waar is; om dit te kunnen checken, moet je a met b vergelijken. 
if (i % 2 === 0) {
  }
In dit geval i % 2 staat voor "a" en 0 staat voor "b".
Je vergelijkt ze met een vergelijkingsoperator === (op waarde en type)